misc/landing-page/src/pages/demos/local-forms.astro

```diff
@@ -0,0 +1,159 @@
+---
+import Layout from "../../layouts/Layout.astro";
+import Section from "../../components/Section.astro";
+---
+
+<Layout>
+  <Section background="light">
+    <header class="mb-4 text-center">
+      <h1
+        class="font-handjet text-brown-header mb-4 text-4xl font-bold md:text-6xl"
+      >
+        Local Forms
+      </h1>
+      <p class="mx-auto max-w-2xl">
+          The following example implements phoenix forms using LocalLiveView.
+      </p>
+      <p class="mx-auto max-w-2xl">
+          The form_demo is built as html page with a single form and a rendered list of users (usernames and emails).
+          List of users is stored in local live view process memory.
+          The form handles onchange and onsubmit LocalLiveView events.
+          The change event triggers validate function that looks through already saved users and makes sure that
+          new username or email isn't already present in the users list. The submit event adds a new user to the user list.
+      </p>
```

Comment:
This could be formatted better with `<code>` elements, and with multiple paragraphs for legibility.